Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.executor.max_memory.count,gauge,,byte,,The number of times the maximum memory used in the application's executors was sampled,0,spark,exe max mem cnt
+spark.rdd.count,gauge,,,,The number of RDDs in the Spark application,0,spark,rdd cnt
+spark.rdd.num_partitions.max,gauge,,,,The max number of partitions in the application's RDDs,0,spark,rdd num part mx
+spark.rdd.num_partitions.avg,gauge,,,,The average number of partitions in the application's RDDs,0,spark,rdd num part avg
+spark.rdd.num_partitions.median,gauge,,,,The median number of partitions in the application's RDDs,0,spark,rdd num part med
+spark.rdd.num_partitions.95percentile,gauge,,,,The 95th percentile number of partitions in the application's RDDs,0,spark,rdd num part 95
+spark.rdd.num_partitions.count,gauge,,,,The number of times the number of partitions in the application's RDDs was sampled,0,spark,rdd num part cnt
+spark.rdd.num_cached_partitions.max,gauge,,,,The max number of cached partitions in the application's RDDs,0,spark,rdd num ch part mx
+spark.rdd.num_cached_partitions.avg,gauge,,,,The average number of cached partitions in the application's RDDs,0,spark,rdd num ch part avg
+spark.rdd.num_cached_partitions.median,gauge,,,,The median number of cached partitions in the application's RDDs,0,spark,rdd num ch part med
+spark.rdd.num_cached_partitions.95percentile,gauge,,,,The 95th percentile number of cached partitions in the application's RDDs,0,spark,rdd num ch part 95
+spark.rdd.num_cached_partitions.count,gauge,,,,The number of times the number of cached partitions in the application's RDDs was sampled,0,spark,rdd num ch part cnt
+spark.rdd.memory_used.max,gauge,,byte,,The max amount of memory used in the application's RDDs,0,spark,rdd mem usd mx
+spark.rdd.memory_used.avg,gauge,,byte,,The average amount of memory used in the application's RDDs,0,spark,rdd mem usd avg
+spark.rdd.memory_used.median,gauge,,byte,,The median amount of memory used in the application's RDDs,0,spark,rdd mem usd med
+spark.rdd.memory_used.95percentile,gauge,,byte,,The 95th percentile amount of memory used in the application's RDDs,0,spark,rdd mem usd 95
+spark.rdd.memory_used.count,gauge,,byte,,The number of times the amount of memory used in the application's RDDs was sampled,0,spark,rdd mem usd cnt
+spark.rdd.disk_used.max,gauge,,byte,,The max amount of disk used in the application's RDDs,0,spark,rdd dsk usd mx
+spark.rdd.disk_used.avg,gauge,,byte,,The average amount of disk used in the application's RDDs,0,spark,rdd dsk usd avg
+spark.rdd.disk_used.median,gauge,,byte,,The median amount of disk used in the application's RDDs,0,spark,rdd dsk usd med
+spark.rdd.disk_used.95percentile,gauge,,byte,,The 95th percentile amount of disk used in the application's RDDs,0,spark,rdd dsk usd 95
+spark.rdd.disk_used.count,gauge,,byte,,The number of times the amount of disk used in the application's RDDs was sampled,0,spark,rdd dsk usd cnt
```

Comment:
For all the `spark.rdd.*` metrics let's specify that these are persisted RDDs, to avoid confusion with other non-persisted RDDs that the application might have.